rosbag2_py/src/rosbag2_py/_info.cpp

```diff
@@ -42,15 +50,29 @@ class Info
     return info_->read_metadata(uri, storage_id);
   }
 
-  void print_output(const rosbag2_storage::BagMetadata & metadata_info)
+  void print_output(
+    const rosbag2_storage::BagMetadata & metadata_info, std::string sorting_method)
```

Comment:
Please use const reference where possible for input parameters.
```suggestion
  void print_output(
    const rosbag2_storage::BagMetadata & metadata_info, const std::string & sorting_method)
```